### DISCUSSION TITLE
Galaxy Brain Achievement

### DISCUSSION THREAD

**Discussion by ghost** [Schweinepriester/github-profile-achievements#320] [General]

**Comment by Schweinepriester** [comment]
It‘s "You need to become the marked answer to 2 discussions?"

**Comment by giovanni060912** [comment]
How do I earn this badge?

**Comment by bacca87** [comment]
just accept my answer, it's easy

**Comment by cccy0** [comment]
come on ~

**Comment by WHAHA-HA** [comment]
Actually, I can achieve Quickdraw and Pull Shark badges with ease, but no Galaxy Brain.
If you want me to do anything, feel free to let me know anytime
PS: It'd be appreciated if you could mark my answer
